recipes/lazyslide/conda-forge.yml

```diff
@@ -0,0 +1,4 @@
+azure:
+  settings_win:
+    variables:
+      SET_PAGEFILE: "True"
```

Comment:
why do you need this? this looks like a pure python package, with no need for extended pagefile during compilation

Reply:
I had put this for debugging. Removed.